src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -46,21 +46,65 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
+   /*
+    * Creates a list of messages posted either by all users if parameter string 'user' is empty or
+    * by one specified user if parameter string is not empty. 
+    * @return a list of messages, or empty list if no messages were posted by any users.
+    */
+
```

Comment:
Don't need this space here. It is better to have the comment right above the function without any spaces.

Reply:
Ok, I will change that right away!